services/mail/pod-inbound-mail/src/message.ts

```diff
@@ -61,13 +61,42 @@ export async function createMessages (
     console.error(`[${mailId}] Unable to create message without a proper FROM`)
     return
   }
+  try {
+    await ensureLocalPerson(
+      client,
+      mailId,
+      fromPerson.uuid,
+      fromPerson.socialId,
+      from.address,
+      fromPerson.firstName,
+      fromPerson.lastName
+    )
+  } catch (err) {
+    console.error(`[${mailId}] Failed to ensure local FROM person`, err)
+    console.error(`[${mailId}] Unable to create message without a proper FROM`)
```

Comment:
[Minor] Isn't it enough to log the error once?

Reply:
This is for generality with global person failure - there will be an error with stack, and then a separate message about the process has been interrupted